### DISCUSSION TITLE
whats the average pay of an secuirty researcher?

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/security-and-auditing-full-course-s23#140] [General]
I know this kind off an off topic question.
but what the average pay of security researcher/auditor. 
do they earn good by mostly contests or auditing firms or protocol hires them privately for reviewing code?

**Comment by EngrPips** [comment]
Hello @cell0901 , honestly i don't have much knowledge about what the pay is as of now and i don't know whether contest pays better or being employed but trust there are people with vast knowledge of this topic , so let's wait to hear from people who knows.

**Comment by funkornaut001** [comment]
On an application for Guardian Audits, the pay for an "entry-level" security researcher was 90-115k but split 50/50 between base pay and bonuses for findings.

**Reply by EngrPips** [reply]
Awesome, good to know this. thanks @funkornaut001
